containers/Overview/index.tsx

```diff
@@ -124,15 +124,17 @@ class Overview extends React.Component<Props> {
           </Box>
         </Box>
         <Box basis={'1/2'} margin={{ left: 'large' }}>
-          <Box>
-            <Heading level="4" margin={{ top: 'small', bottom: 'small' }}>Asset Originator Details</Heading>
-
-            <Text margin={{ top: 'xsmall' }}> {name} </Text>
-            <Text margin={{ top: 'medium' }}> The following information is provided by the Asset Originator: </Text>
+          <div>
+            <Heading level="4" margin={{ top: 'small', bottom: 'small' }}>The following information is provided by the Asset Originator</Heading>
 
             <div dangerouslySetInnerHTML={{ __html: description }} />
 
-          </Box>
+            {details && <Heading level="4" margin={{ top: 'small', bottom: 'small' }}>Pool Details</Heading>}
+
+            {details && <div dangerouslySetInnerHTML={{ __html: details }} />}
+
+            <p>Want to invest in this Tinlake pool? <a href="http://centrifuge.io/invest" target="_blank">Learn how...</a></p>
```

Comment:
@philipstanislaus can you remove this hardcoded link? it will be added in the details.